openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -101,6 +116,25 @@ paths:
           $ref: '#/components/responses/PermissionDenied'
         '500':
           $ref: '#/components/responses/ServerError'
+    get:
+      parameters:

```

Comment:
Looks like the `LandingZoneId ` parameter is already defined on L97 so maybe not needed here?

Reply:
It actually worked - good to know that parameters are shared despite the syntaxis.

Reply:
Yes, the resulting `openapi.yaml` file is flat, so anything can reference anything. I had some ideas for scoping access so that we wouldn't end up with a bowl of spaghetti of references, but it got too complicated.

If you have ideas for improving the structure of that source, I'd be interested.